src/State/State.js

```diff
@@ -69,10 +76,12 @@ class State extends Component {
           icon="import-export"
           renderActions={() => (
             <div>
-              <IconAddBackup
+              <Button
+                icon="file-download" 
+                onClick={stateBackupStore.sendBackup()} 
```

Comment:
This will execute the function at render time. We need to drop the `()` so we pass the function instead